PyISY/Nodes/group.py

```diff
@@ -21,11 +23,12 @@ class Group(object):
     status = Property(0)
     hasChildren = False
 
-    def __init__(self, parent, nid, name, members=[]):
+    def __init__(self, parent, nid, name, members=[], controllers=[]):
         self.parent = parent
         self._id = nid
         self.name = name
         self._members = members
+        self._controllers = controllers
```

Comment:
Then this line would be something like:

```python
self._controllers = controllers or []
```

The reason for this can be seen [here](http://docs.python-guide.org/en/latest/writing/gotchas/#mutable-default-arguments).